src/chat/components/bubble/CustomBubble.tsx

```diff
@@ -30,25 +32,42 @@ export const CustomBubble: React.FC<CustomBubbleProps> = ({
   customTextStyle,
   unReadSeenMessage,
   unReadSentMessage,
+  disableSeenMessage,
+  customUnReadView,
 }) => {
   const styleBuble = {
     left: { backgroundColor: 'transparent' },
     right: { backgroundColor: 'transparent' },
   };
 
-  const renderBubble = (currentMessage: MessageProps) => {
-    const isMyLatestMessage =
-      !Object.keys(bubbleMessage.nextMessage ?? {}).length &&
-      position === 'right';
-    const ViewRead = isMyLatestMessage && (
+  const viewUnRead = (
```

Comment:
```suggestion
  const renderViewUnRead = (
```
Rename this for clarification